x/distribution/types/codec.go

```diff
@@ -12,11 +12,11 @@ func RegisterCodec(cdc *codec.Codec) {
 }
 
 // generic sealed codec to be used throughout module
-var MsgCdc *codec.Codec
+var ModuleCdc *codec.Codec
```

Comment:
Why does the codec have to be public?

Reply:
because it's also used by other subpackages within the module (it's sealed though!)